src/util.h

```diff
@@ -572,6 +572,20 @@ enum ECSAddType
 };
 
 
+// Registration status ---------------------------------------------------------
+enum ESvrRegStatus
+{
+    SRS_BAD_ADDRESS = -4,
```

Comment:
Please use just positive numbers starting from 0. This would be consistent with the existing enums used for the protocol.

Reply:
The negative values aren't part of the protocol, that's why I used negative.  Only the the non-negative values are exchanged at the protocol layer.

Reply:
Thinking about it more, the two separate enums would probably be more logical - they represent two different things:
- central server outcome of request
- slave server registration status (which may not even include registering)